apsuite/optics_analysis/base_correction.py

```diff
@@ -0,0 +1,188 @@
+"""."""
+
+from collections import namedtuple as _namedtuple
+from copy import deepcopy as _dcopy
+import numpy as np
+
+__KNOBTYPES = _namedtuple('KnobTypes', ['Focusing', 'Defocusing'])
+
+
+class KnobTypes(__KNOBTYPES):
```

Comment:
Pylint complains about namedtuple inheretence (`inherent-non-class`)
In my opinion we should avoid it and should use simpler mechanisms.

https://stackoverflow.com/questions/42385916/inheriting-from-a-namedtuple-base-class-python
http://zecong.hu/2019/08/10/inheritance-for-namedtuples/

Reply:
@fernandohds564

Reply:
ok I agree! I was thinking about changing this. We could create a simple class with object attributes `focusing` and `defocusing` and a property named `all` which combines both.

Reply:
done!